components/backup-stream/src/endpoint.rs

```diff
@@ -126,6 +127,7 @@ where
 
         let initial_scan_memory_quota =
             PendingMemoryQuota::new(config.initial_scan_pending_memory_quota.0 as _);
+        let initial_scan_throughput_quota = Limiter::new(config.initial_scan_rate_limit.0 as _);
```

Comment:
if not set `config.initial_scan_rate_limit`, we should put it  `f64::INFINITY`

Reply:
Maybe need fill it in Default()